account_reversal/README.rst

```diff
@@ -1,23 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
 Account Reversal
 ================
 
 This module adds an action "Reversal" on account moves,
 to allow the accountant to create reversal account moves in 2 clicks.
 Also add on account entries:
 
- * a checkbox and filter "to be reversed"
- * a link between an entry and its reversal entry
+* a checkbox and filter "to be reversed"
+* a link between an entry and its reversal entry
+
+Odoo v9c include a similar action (overwritten by this addon), but with less
+features, for instance:
+
+* Allowing inheritance
+* Options like prefix (for journal entry and journal item), post and reconcile.
+* Create a link between the entry and its reversal
+* Mark entries to be reversed in the future.
+
+Usage
+=====
+
+As in Odoo v9c, if you select an entry from Accounting > Adviser > Journal Entries,
+then an action menu 'Reverse Entries' is available. If clicked, then a wizard
+allows user to select Reversal Date, Reversal Journal, Prefix, Post and Reconcile.
+
+* If no Reversal Journal is selected, then the same journal is used
+* If Post is True, then reversal entry will be posted else it will be leaved
+as a draft entry
+* If Post and Reconcile are True, then all entry lines with reconciled accounts
+of the entry will be reconciled with the reserval entry ones.
+
+Also, there is new menu Accounting > Adviser > Journal Entries to be Reversed
```

Comment:
There is also a new menu Accounting > Adviser > Journal Entries to be Reversed
in order to allow tracking entries that must be reserved for any reason.